bigchaindb_driver/driver.py

```diff
@@ -122,14 +122,12 @@ def create(self, payload=None, verifying_key=None, signing_key=None):
         verifying_key = verifying_key if verifying_key else self.verifying_key
         if not verifying_key:
             raise InvalidVerifyingKey
-        fulfillment = Fulfillment.gen_default([verifying_key])
-        condition = fulfillment.gen_condition()
-        data = Data(payload=payload)
-        transaction = Transaction(
-            'CREATE',
-            fulfillments=[fulfillment],
-            conditions=[condition],
-            data=data,
+        # TODO: In the future, we should adjust this to let the user of the
+        #       driver define both `owners_before` and `owners_after`.
+        transaction = Transaction.create(
+            owners_before=[verifying_key],
```

Comment:
Does `owners_before` make sense when creating transactions? Is it necessary (shouldn't the node that creates the transaction fill that in)?